_pages/en/strategy-communications.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+title: Communications Strategy
+ref: strategy-communications
+lang: en
+status: posted
+sections: Work In Progress
+permalink: /strategy-communications.html
+---
+
+## Communications Strategy
```

Comment:
Add Heading "Purpose"

To elaborate on the re-usable methods the IT Strategy teams will have at their disposal in communicate effectively its strategies. The goals of the communication strategy are:

1. Raise awareness of IT Strategies being produced and approved to ESDC personnel, more specifically IITB ones
2. Engage effectively with stakeholders during both the strategy making process and their execution
3. Demonstrate the success of IT strategies
4. Ensure stakeholders understand the intent of the IT strategies
5. Change behaviours and perceptions where necessary